app/util/bitbucket/upload_attachments.sh

```diff
@@ -6,7 +6,7 @@ pgrep nfsd > /dev/null && echo "NFS found" || { echo NFS process was not found.
 ###################    Variables section         ###################
 # Bitbucket version variables
 BITBUCKET_VERSION_FILE="/media/atl/bitbucket/shared/bitbucket.version"
-SUPPORTED_BITBUCKET_VERSIONS=(6.10.5 7.0.5 7.6.0)
+SUPPORTED_BITBUCKET_VERSIONS=(7.6.0 7.0.5 6.10.5)
```

Comment:
do not change the order here, this order is used in by `force` flag to identify the latest version.

Reply:
Returned back